bambi/priors.py

```diff
@@ -244,31 +246,33 @@ def _get_slope_stats(self, exog, predictor, sd_corr, full_mod=None, points=4):
                 for val in values[:-1]
             ]
             null = np.append(null, full_mod)
-            ll = np.array([x.llf for x in null])
+            approximation = np.array([x.llf for x in null])
```

Comment:
We could use log_likelihood instead of approximation

Reply:
b419ef5